### PR TITLE
chore: dep updates

### PR DESCRIPTION
**Pull request by Miradorn**
update all deps, bumped nimble_parsec to next minor

### CODE REVIEW

File: mix.exs [hunk 1]
```diff
@@ -44,7 +44,7 @@ defmodule MakeupElixir.Mixfile do
   defp deps do
     [
       {:makeup, "~> 1.0"},
-      {:nimble_parsec, "~> 1.2.3"},
+      {:nimble_parsec, "~> 1.3"},
```
Comment (josevalim) [inline mix.exs:47]:
```suggestion
      {:nimble_parsec, "~> 1.2.3 or ~> 1.3"},
```

**Comment by josevalim** [comment]
:green_heart: :blue_heart: :purple_heart: :yellow_heart: :heart:

**Comment by Miradorn** [comment]
thanks for merging @josevalim ❤️ would you mind cutting a new release? this blocks us from upgrading a transitive dep in our codebase

**Comment by josevalim** [comment]
done!

**Pull request merged**